juju/docs
# Add navigation metadata

## QA
_if you want to_
- Run `documentation-builder --source-folder src --output-media-path 'static/media' --media-url '/static/media' --tag-manager-code 'GTM-K92JCQ' --search-domain 'docs.jujucharms.com' --search-url 'https://docs.ubuntu.com/search' --search-placeholder 'Search Juju docs' --no-link-extensions`
- Check out build/en/charms-scaling.html
- Navigate around by adding `.html` to the end of the urls
